### PR TITLE
Halves the time needed to hack a door with the access tuner

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Halves the use time to hack a door with  the access tuner.

## Why It's Good For The Game
When was the last time you've seen someone use an access tuner? 
Currently, it's not bought a lot as an emag is usually just better, works instantly, can be used while running away, can have functions beyond a door opener and the only downside of it is that the door is bolted open for both sec and themselves. 
This PR should give the access tuner some more bang for their buck, as it is currently priced as 6 TC, same as the emag.

## Testing
Walked around, could actually hack a good amount of doors in less time.

## Changelog
:cl:
tweak: The access tuner hack time is halved
/:cl:

### CODE REVIEW

File: code/game/objects/items/control_wand.dm [hunk 1]
```diff
@@ -138,7 +138,7 @@
 	desc = "A device used for illegally interfacing with doors."
 	icon_state = "hacktool"
 	item_state = "hacktool"
-	var/hack_speed = 30
+	var/hack_speed = 15
```
Comment (GDNgit) [inline code/game/objects/items/control_wand.dm:141]:
```suggestion
	var/hack_speed = 1.5 SECONDS
```
Also your pr cl suggests it's been doubled, like to 6 seconds, might want to rephrase

**Review by GDNgit — CHANGES_REQUESTED** [review]

File: code/game/objects/items/control_wand.dm (hunk 1 above)
Comment (DGamerL) [reply code/game/objects/items/control_wand.dm:141]: Oh yeah that's just a conflict of saying it easily for the non-coders and the var name, will clarify

**Review by Burzah — APPROVED** [review]

**Review by Henri215 — APPROVED** [review]

**Comment by tipoima** [comment]
I feel like you're undervaluing the tuner. While it's slow, it's infinitely stealthier than emag and infinitely less gibbing than the teleporter. You also can't emag through cameras or if the door is shocked.
I'm not _necessarily_ saying it shouldn't be sped up, but should it?

**Review by lewcc — APPROVED** [review]
Code looks good

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**